api2/pkg/relationshipparties/static.go

```diff
@@ -0,0 +1,24 @@
+package relationshipparties
+
+import (
+	"context"
+	"github.com/nrc-no/core-kafka/pkg/parties/attributes"
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/mongo"
+)
+
+func Init(ctx context.Context, partiesStore *PartiesStore) error {
+	if _, err := partiesStore.store.Collection.Indexes().CreateOne(ctx, mongo.IndexModel{
```

Comment:
I would move this to individuals/static.go or parties/static.go (this is not really a concern of relationshipparties, but rather of individuals)